content/applications/general/email_communication/email_servers.rst

```diff
@@ -1,222 +1,210 @@
 
-* Create the corresponding email addresses in your mail server 
-  (catchall@, bounce@, sales@, etc.).
-* Set your domain name in the General Settings.
+* Create the corresponding email addresses in the mail server (catchall@, bounce@, sales@, etc.).
+* Set the :guilabel:`Alias Domain` name in :menuselection:`Settings --> General Settings -->
+  Discuss`.
 
-  .. image:: email_servers/alias-domain.png
-      :align: center
+* If the database's hosting type is Odoo on-premise, create an :guilabel:`Incoming Mail Server` in
+  Odoo for each alias. This can be done from the General Settings as well. Fill out the form
+  according to the email provider's settings. Leave the :guilabel:`Actions to Perform on Incoming
+  Mails` field blank. Once all the information has been filled out, click on :guilabel:`TEST &
+  CONFIRM`.
 
-* If you use Odoo on-premise, create an *Incoming Mail Server* in Odoo for each alias. 
-  You can do it from the General Settings as well. Fill out the form according 
-  to your email provider’s settings. 
-  Leave the *Actions to Perform on Incoming Mails* blank. Once all the 
-  information has been filled out, click on *TEST & CONFIRM*.
+  .. image:: email_servers/incoming-server.png
+     :align: center
+     :alt: Incoming mail server configuration on Odoo.
 
-.. image:: email_servers/incoming-server.png
-    :align: center
+* If the database's hosting type is Odoo Online or Odoo.sh, redirecting incoming messages to Odoo's
+  domain name instead of the external email server is recommended. That way, incoming messages can
+  be received without delay. Odoo Online only fetches incoming messages of external servers once
+  every hour. Redirections for all email addresses should be set to Odoo's domain name in the email
+  server (e.g. *catchall\@mydomain.ext* to *catchall\@mycompany.odoo.com*).
 
-* If you use Odoo Online or Odoo.sh, We do recommend to redirect incoming messages 
-  to Odoo's domain name rather than exclusively use your own email server. 
-  That way you will receive incoming messages without delay. Indeed, Odoo Online is fetching
-  incoming messages of external servers once per hour only. 
-  You should set redirections for all the email addresses to Odoo's domain name in your 
-  email server (e.g. *catchall@mydomain.ext* to *catchall@mycompany.odoo.com*).
+All the aliases are customizable in Odoo. Object aliases can be edited from their respective
+configuration view.
 
-.. tip:: All the aliases are customizable in Odoo.
-   Object aliases can be edited from their  respective configuration view.
-   To edit catchall and bounce aliases, you first need to activate the
-   :ref:`developer mode <developer-mode>`.
+.. tip:: To edit catchall and bounce aliases, first activate the :ref:`developer mode
+   <developer-mode>`. Then, go to :menuselection:`Settings --> Technical --> Parameters --> System
+   Parameters` to customize the aliases (*mail.catchall.alias* & *mail.bounce.alias*).
```

Comment:
Nitpicking (i.e. no need to change that here, but take into account for the next PRs): it's better to start the content of tips/notes on the next line, otherwise it's interpreted as the tip/note's title, which is not supported.
```suggestion
.. tip::
   To edit catchall and bounce aliases, first activate the :ref:`developer mode
   <developer-mode>`. Then, go to :menuselection:`Settings --> Technical --> Parameters --> System
   Parameters` to customize the aliases (*mail.catchall.alias* & *mail.bounce.alias*).
```